store/utils.js

```diff
@@ -74,6 +79,10 @@ export const initMiddleware = () => {
   return camelcaseKeysDeep(swg.api)
 }
 
+export const loadNextPage = async (nextPageUrl) => {
+  return camelcaseKeysDeep(await fetchJson(`${process.env.middlewareURL}/${nextPageUrl}`))
```

Comment:
```suggestion
export const fetchMiddleware = async (path) => {
  return camelcaseKeysDeep(await fetchJson(`${process.env.middlewareURL}/${path}`))
```
it doesn't know anything about the next page 🤷‍♀️